### PR TITLE
closes #75

### LINKED ISSUE #75: Tidy up README.

### PR DESCRIPTION
**Pull request by liamtlr**
A bit wordy/patronising, any comments welcome ...

Purposefully didn't include many screenshots in case any of us gets around to doing the last fine formatting tweaks in the CSS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,143 @@
...
+ * Search the properties listed on Makersbnb
+ * Or, for the indecisive, view all listings
+
+### Searching Makersbnb
+
+Searching can be done using a unique listing id for each property or a specific city. In addition, you can enter the dates you'd like, so that you only get relevant options.
+
+### Requesting a site
+
+For each listing, you'll see  the dates for which it's available. If you've specified dates in tyour search, you'll only see options within those dates.
```
Comment (hanrattyjen) [inline README.md]: Just a typo in 'tyour' on line 52.

**Review by hanrattyjen — COMMENTED** [review]
Looks great! Just that one typo. Didn't want to confirm PR until Glenn and James had a look!

File: README.md [hunk 2]
```diff
@@ -1,10 +1,143 @@
...
+Please note, currently you can only request stays based on the slots specified by the host. We're working on improving the site by allowing you to suggest your own dates.
+
+### Listing a property
+
+Listing a property allows you to enter the details of properties you'd like to make available for other users of the site. It's pretty straightforward, just click the option in the navigation bar and enter the required details. Once all this is done, you'll get a message confirming the listing has been successfully added.
+
+### Adding available dates
+
+Once you've added a property, you'll be prompted to specify when it's available so that users can request that slot. Simply click the relevant link and enter the dates. To enter further dates, just go to you profile and select the 'Add dates' option next to this listing.
+
```
Comment (glynester) [inline README.md]: "just go to you profile" - should be "your"

Comment (glynester) [inline README.md]: To do this, go to **you** profile page - and below same thing.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,143 @@
...
+### Approving a request
+
+So your listing is live on the site, and you've entered when it's available, now you're just waiting for it to be requested by another user. Once this has happened, you will receive an email. You can now approve the request (or reject it, if you're feeling mean).
+
+To do this, go to you profile page. In the field, 'Received Requests', you'll see all the requests for your property listings, which you can individually accept or reject. Once a request has been accepted, it will be removed from the 'avialable dates' for that listing.
+
```
Comment (glynester) [inline README.md]: avialable - should be available

File: README.md [hunk 4]
```diff
@@ -1,10 +1,143 @@
-[![Stories in Ready](https://badge.waffle.io/James-SteelX/makersbnb.png?label=ready&title=Ready)](http://waffle.io/James-SteelX/makersbnb)
```
Comment (glynester) [inline README.md]: Great job Liam. Liked the humour too!

File: README.md (hunk 1 above)
Comment (hanrattyjen) [inline README.md]: I missed all of these!!

**Comment by wirsindpapst** [comment]
Updated based on your feedback .... thanks guys!
